openapi-diff/src/modeler/AutoRest.Swagger/ComparisonMessages.cs

```diff
@@ -345,5 +353,13 @@ public static class ComparisonMessages
             Message = "The new version has a different '{0}' value than the previous one.",
             Type = MessageType.Update
         };
+
+        public static MessageTemplate LongRunningOperationExtensionChanged = new MessageTemplate
+        {
+            Id = 1036,
```

Comment:
Should be 1044

Reply:
BTW, how can you found this issue ?

Reply:
I guess this issue is visible to everyone :)